Packs/Core/Playbooks/playbook-WildFire_Malware.yml

```diff
@@ -1727,7 +1758,12 @@ inputs:
   value:
     complex:
       root: alert
-      accessor: initiatorsha256
+      transformers:
+      - operator: DT
+        args:
+          dt:
+            value:
+              simple: .=pickvalue(val);function pickvalue(x){if(x.initiatorsha256){return x.initiatorsha256} else {return x.filesha256}}
```

Comment:
BTW, can we use if-then-else here as we did in the query input, or even shorten if-then-else like 
(statement)? <what will be the true value> ; <what will be the true value>

Reply:
we sure can, but the DT if statement is more accurate here.